README.md

```diff
@@ -28,17 +28,47 @@ DataPull is a self-service Distributed ETL tool to join and transform data from
   cd ./datapull
   docker build -f ./core/docker_spark_server/Dockerfile -t expedia/spark2.4.8-scala2.11-hadoop2.10.1 ./core/docker_spark_server
   ```
-* build the Scala JAR from within the core folder
+* Build the Scala JAR from within the `core` folder
   ```shell script
   cd ./core
   cp ./src/main/resources/application-dev.yml ./src/main/resources/application.yml
-  docker run -e MAVEN_OPTS="-Xmx1024M -Xss128M -XX:MetaspaceSize=512M -XX:MaxMetaspaceSize=1024M -XX:+CMSClassUnloadingEnabled" --rm -v "${PWD}":/usr/src/mymaven -v "${HOME}/.m2":/root/.m2 -w /usr/src/mymaven maven:3.6.3-jdk-8 mvn clean install
   ```
-* Execute a sample JSON input file [Input_Sample_filesystem-to-filesystem.json](core/src/main/resources/Input_Sample_filesystem-to-filesystem.json) that moves data from a CSV file [HelloWorld.csv](core/src/main/resources/SampleData/HelloWorld/HelloWorld.csv) to a folder of json files named SampleData_Json.  
+  Edit the file `./src/main/resources/application.yml` to specify the `inputjson:` value, for example:
```

Comment:
This step isn't really required, imho.

Reply:
I replaced the `cp` step with the appropriate file:
```shell
cp ../master_application_config-dev.yml ./src/main/resources/application.yml
```